File: fontes/backend/node/retaguarda-sh/src/alexa/alexa-response.ts
import { AlexaOutputSpeech } from "./alexa-output-speech";

export class AlexaResponse {

	shouldEndSession: boolean;
    outputSpeech: AlexaOutputSpeech;

	/**
	* Constructor
	*/
	constructor(objetoJson: object) {
		if (objetoJson != null) {
			this.shouldEndSession = objetoJson['shouldEndSession'];
			this.outputSpeech = objetoJson['outputSpeech'];
		}	
	}

}

File: fontes/backend/java/retaguarda_sh/src/main/java/com/t2ti/retaguarda_sh/model/alexa/AlexaResponse.java
package com.t2ti.retaguarda_sh.model.alexa;

import java.io.Serializable;

public class AlexaResponse implements Serializable {
	private static final long serialVersionUID = 1L;
	
	private boolean shouldEndSession;
	private AlexaOutputSpeech outputSpeech;

	public AlexaResponse() {}

	public boolean isShouldEndSession() {
		return shouldEndSession;
	}

	public void setShouldEndSession(boolean shouldEndSession) {
		this.shouldEndSession = shouldEndSession;
	}

	public AlexaOutputSpeech getOutputSpeech() {
		return outputSpeech;
	}

	public void setOutputSpeech(AlexaOutputSpeech outputSpeech) {
		this.outputSpeech = outputSpeech;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}


}